wpilibj/athena.gradle

```diff
@@ -114,6 +108,44 @@ task wpilibjSources(type: Jar, dependsOn: classes) {
     from sourceSets.shared.allJava
 }
 
+task wpilibJNativeLibraries(type: Jar) {
+    description = 'Creates the native library jar for the maven publishing routine'
+    group = 'WPILib'
+    classifier = 'nativelibraries'
```

Comment:
We'll still want to set the classifier here, so the name of the binary in the build tree is correct. We'll just set it in the publishing section.